b2sdk/v1/account_info.py

```diff
@@ -76,7 +79,24 @@ class UrlPoolAccountInfo(v2.UrlPoolAccountInfo, AbstractAccountInfo):
 
 
 class SqliteAccountInfo(v2.SqliteAccountInfo, AbstractAccountInfo):
-    pass
+    def __init__(self, file_name=None, last_upgrade_to_run=None):
+        """
+        If ``file_name`` argument is empty or ``None``, path from ``B2_ACCOUNT_INFO`` environment variable is used. If that is not available, a default of ``~/.b2_account_info`` is used.
+
+        :param str file_name: The sqlite file to use; overrides the default.
+        :param int last_upgrade_to_run: For testing only, override the auto-update on the db.
+        """
+        # use legacy env var resolution, XDG not supported
+        self.thread_local = threading.local()
+        user_account_info_path = file_name or os.environ.get(
+            v2.B2_ACCOUNT_INFO_ENV_VAR, v2.B2_ACCOUNT_INFO_DEFAULT_FILE
+        )
+        self.filename = file_name or os.path.expanduser(user_account_info_path)
+        logger.debug('%s file path to use: %s', self.__class__.__name__, self.filename)
```

Comment:
where is `logger` initialized?

Reply:
It's imported from here:

https://github.com/Backblaze/b2-sdk-python/blob/632b13b74aa584a9c962a9b85f4630b8373cba1d/b2sdk/account_info/sqlite_account_info.py#L23

Reply:
ok, but here in `b2sdk/v1/account_info.py` there is no `logger`, so it will crash. Logger initialization should be added.

Reply:
Sorry, bad wording. The initialization happens there, and there's an import in this file *for* that logger.

It probably should get initialized here anyway, though.

Reply:
`from b2sdk.account_info.sqlite_account_info import logger` ohhh. I don't think we should do that, that is super confusing. We should initialize it locally, right?